_episodes/06-files.md

```diff
@@ -13,7 +13,7 @@ keypoints:
 ---
 
 We now have almost everything we need to process all our data files.
-The only thing that's missing is a library with a rather unpleasant name:
+The only thing that's missing is a library which provides functionality for filtering files based on their naming scheme:
 
```

Comment:
I think starting the clause with "that" instead of "which" would be proper. Otherwise, I like the wording!

Reply:
This explanation seems a tiny bit too complex, plus not all files have a naming scheme and `glob` can also return directories.

I suggest to rewrite this sentence and don't talk about a library that is missing but functionality that we need. And what we need is a way to get a list of files from the `data` directory whose names begin with `inflammation-` and end with `.csv`. What do you think?

Reply:
I think that works too @maxim-belkin  and would be more directly to the point. Perhaps something along the lines of replacing this sentence and the one above it with:

_As a final piece to processing our inflammation data, we need  a way to get a list of all the files in our `data` directory whose names start with `inflammation-` and end with `.csv`._

Reply:
That indeed sounds good! I just feel that now a small sentence is missing to actual make the link to the import statement that is following. Could we continue above sentence with "The following library will help us to achieve this"?

Reply:
That's a good point @codingS3b.  I think that language makes sense.